pkg/shell/shell.css

```diff
@@ -601,6 +614,12 @@ ul.available-interfaces-group {
     display: none !important;
 }
 
+/* Grow the delete icon a bit in order to make it render better */
+#containers-containers .btn-delete,
+#image-details-containers .btn-delete {
```

Comment:
I was kind of vary about changing that all across the board, but since we need to fix this close to delete icon in other places as well, might as well do it for all of them.